### PR TITLE
docs(host): add hint for remote consle

### PR DESCRIPTION
**Pull request by WebberHuang1118**
Related to https://github.com/harvester/harvester/issues/5679

### CODE REVIEW

File: docs/host/host.md [hunk 1]
```diff
@@ -533,4 +533,24 @@ status:
 ```
 
 The `harvester-node-manager` pod(s) in the `harvester-system` namespace may also contain some hints as to why it is not rendering a file to a node.
-This pod is part of a daemonset, so it may be worth checking the pod that is running on the node of interest.
\ No newline at end of file
+This pod is part of a daemonset, so it may be worth checking the pod that is running on the node of interest.
+
+## Remote Console
+
+User can set console URL for remote server management, which will be be needed especially in environments where physical access might be limited
```
Comment (a110605) [inline docs/host/host.md:540]: double `be`

**Comment by a110605** [comment]
Could you update the `static/img/remote_console_button.png` with latest UI ? Thanks

File: docs/host/host.md (hunk 1 above)
Comment (WebberHuang1118) [reply docs/host/host.md]: Updated per suggestion, thanks.

**Comment by WebberHuang1118** [comment]
> Could you update the `static/img/remote_console_button.png` with latest UI ? Thanks

GUI seems the same if the URL is configured
Screenshot_20241018_151031

**Review by bk201 — APPROVED** [review]

**Review by a110605 — APPROVED** [review]
LGTM

File: docs/host/host.md [hunk 2]
```diff
@@ -533,4 +533,24 @@ status:
...
+
+1. Specify **Console URL** for remote server management
+
+  Following is an example with HP iLO:
+
+  ![](/img/remote_console_url.png)
+
+1. Click **Console** to access remote server
+
+  ![](/img/remote_console_button.png)
```
Comment (jillian-maroket) [inline docs/host/host.md:556]:
```suggestion
You can configure the URL of the console for remote server management. This console is particularly useful in environments where physical access is limited.

1. On the Harvester UI, go to **Hosts**.

1. Locate the target host, and then select **⋮ > **Edit Config**.

  ![](/img/remote_console_config.png)

1. Specify the **Console URL**, and then click **Save**.

  Example (with HPE iLO):

  ![](/img/remote_console_url.png)

1. Click **Console** to access the remote server.

  ![](/img/remote_console_button.png)
```

**Review by jillian-maroket — COMMENTED** [review]
Looks good overall. I just cleaned up the wording.

File: docs/host/host.md (hunk 2 above)
Comment (WebberHuang1118) [reply docs/host/host.md]: Updated per suggestion, thanks.

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**